google-cloud-tools-plugin/google-cloud-core/build.gradle

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Use IntelliJ src structure for now
+sourceSets.main.java.srcDirs = ['src']
+sourceSets.main.resources.srcDirs = ['resources']
+sourceSets.test.java.srcDirs = ['testSrc']
+sourceSets.test.resources.srcDirs = ['testResources']
+
+dependencies {
+    compile(project(':common-lib'),  project(':google-cloud-tools-plugin:google-account'))
+    compile 'com.google.apis:google-api-services-cloudresourcemanager:v1beta1-rev12-1.21.0'
+    compile('com.google.cloud:google-cloud-storage:1.2.0') {
```

Comment:
just so I understand: GCS is in here because its common to all IDEs (why not in the GCS vertical)? Do we still need the dependency definition thats in `google-cloud-tools-plugin`?

Reply:
good point. that's from a branch further down the road.  I'm going to remove them until I need them.

Reply:
fixed.